lib/Tab.js

```diff
@@ -133,8 +156,9 @@ export default class Tab extends Component {
           { opacity: this.state[mode].iconOpacity }
         ]}
       >
+
         <View ref="_bnic" collapsable={false}>
-          {active && activeIcon ? activeIcon : icon}
+          {this._renderBadge()}
```

Comment:
_For android compatibility:_ Change back to `{active && activeIcon ? activeIcon : icon}`. Move `{this._renderBadge()}` to the render function, right above `{this._renderIcon()}`

Reply:
done. moved it below it since moving it above was causing problems on iOS.